api/web3server_utils.go

```diff
@@ -117,13 +117,16 @@ func (svr *web3Handler) getTransactionFromActionInfo(blkHash hash.Hash256, selp
 	if err != nil {
 		return nil, err
 	}
+	tx, err := action.RawTxToSignedTx(ethTx, svr.coreService.ChainID(), selp.Signature())
+	if err != nil {
+		return nil, err
+	}
 	return &getTransactionResult{
 		blockHash: blkHash,
 		to:        to,
-		ethTx:     ethTx,
+		ethTx:     tx,
 		receipt:   receipt,
 		pubkey:    selp.SrcPubkey(),
-		signature: selp.Signature(),
```

Comment:
why could signature be deleted?

Reply:
not needed anymore, now the signed tx is generated, which contains correct signature value